### PR TITLE
Bugfix/live 14015 tezos unrevealed

### PR DESCRIPTION
**Pull request by qperrot**
### ✅ Checklist

- [x] `npx changeset` was attached.
- [x] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - ...

### 📝 Description

Try to send funds (using send max) from an unrevealed account to another XTZ account use to fail because of fees too low.
Fix it with a buffer and removed deprecated method from taquito.
Add tests for send full balance from an unrevealed account and to an account with zero balance. 

### ❓ Context

- **JIRA or GitHub link**: https://ledgerhq.atlassian.net/browse/LIVE-14015 

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

**Review by hzheng-ledger — DISMISSED** [review]
looks good

**Review by Wozacosta — DISMISSED** [review]

File: libs/coin-modules/coin-tezos/src/logic/estimateFees.ts [hunk 1]
```diff
@@ -99,8 +99,15 @@ export async function estimateFees({
         throw new UnsupportedTransactionMode("unsupported mode", { mode: transaction.mode });
     }
 
+    // NOTE: if useAllAmount is true, is it for sure in the send mode (ie. transfer)?
```
Comment (Wozacosta) [inline libs/coin-modules/coin-tezos/src/logic/estimateFees.ts:102]: did you check?

File: libs/coin-modules/coin-tezos/src/bridge/prepareTransaction.test.ts [hunk 2]
```diff
@@ -90,31 +91,98 @@ describe("prepareTransaction", () => {
...
+      fees: new BigNumber(tezosEstimate.suggestedFeeMutez),
+      estimatedFees: new BigNumber(tezosEstimate.suggestedFeeMutez),
+      gasLimit: new BigNumber(tezosEstimate.gasLimit),
+      storageLimit: new BigNumber(tezosEstimate.storageLimit),
+      amount: maxAmount.minus(gasLimit - (tezosEstimate.opSize + gasLimit * 0.1)),
+    });
+  });
+
+  it("returns new transaction with estimated fees by TezosToolkit when useAllAmount from unrevealed account to revealed account", async () => {
+    // Given
```
Comment (Wozacosta) [inline libs/coin-modules/coin-tezos/src/bridge/prepareTransaction.test.ts:154]: Are the comments here (and below) necessary?

**Review by Wozacosta — APPROVED** [review]

**Review by Wozacosta — APPROVED** [review]
